python/dask_cudf/dask_cudf/core.py

```diff
@@ -434,6 +419,78 @@ class Index(Series, dd.core.Index):
     _partition_type = cudf.Index
 
 
+def _naive_var(ddf, meta, skipna, ddof, split_every, out):
+    num = ddf._get_numeric_data()
+    x = 1.0 * num.sum(skipna=skipna, split_every=split_every)
```

Comment:
Naive question: What does the `1.0 *` do for us here?

Reply:
My guess (though Rick please feel free to correct me) this is needed to cast to a `float`.

Reply:
Good question :)

This `_naive_var`funciton is a copy-paste of the original code, so I didn't actually write it.  However, my assumption was (as John suggested) that the `1.0` is meant to ensure that all results are cast to `float`.

Reply:
Is this actually a single value? Suspect (though don't know) this is probably a Series or DataFrame, in which case calling `float(...)` on it won't work

Reply:
Ah, sorry; then in that case I'd advocate for a more explicit dtype conversion. Regardless, I'm not hung up on this for this particular PR. It's the sort of thing I'd insist on for new code, but I'm not too fussed about a copy-paste; it can always be cleaned up in a specific code clean-up PR.

Reply:
Yeah unclear if a scalar is also a possibility, in which case this is a little handy for allowing us to succinctly cover all those cases, but am not attached to the logic here either (and can see the argument for being more explicit)

Anyways sounds like we can table this for now. Would you like to open an issue and we can revisit later?